project1/application.py

```diff
@@ -83,12 +84,30 @@ def sessiontimeout():
     session.pop("email",None)
     return redirect(url_for('register'))
 
-@app.route("/user")
+@app.route("/user",methods=["GET","POST"])
 def user():
-    if session.get("email") is not None:
-        return render_template("user.html")
-    return redirect(url_for('register'))
-
+    if session.get("email") is None:
+         return redirect(url_for('register'))
+    if (request.method == "POST"):
+        search = request.form.get("search")
+        print(search)
+        search_query="%"+search+"%"
+        
+        books = Book.query.filter(or_(Book.title.like(search_query), Book.author.like(search_query), Book.isbn.like(search_query))).all()
+        print(books)
+        c=0
+        for each in books:
+            c=c+1
+        print(c)

```

Comment:
@Balaji751 Did you count the number of books here?? If so you could do it by .length() function as query method returns a list.